astroquery/mpc/core.py

```diff
@@ -0,0 +1,294 @@
+from ..query import BaseQuery
```

Comment:
Since this is a UTF-8 file it should have this header (I think this is only needed for Python 2).
```
# -*- coding: utf-8 -*-
```